plugin.json

```diff
@@ -11,14 +11,26 @@
       "windows-amd64": "server/dist/plugin-windows-amd64.exe"
     }
   },
+  "webapp": {
+    "bundle_path": "webapp/dist/main.js"
+  },
   "settings_schema": {
-    "settings": [{
-     "key": "Trigger",
-     "display_name": "Trigger Word",
-     "type": "text",
-     "help_text": "Trigger Word must be unique, and cannot begin with a slash or contain any spaces.",
-     "default": "poll"
-     }],
-     "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/matterpoll/matterpoll)."
+    "settings": [
+      {
```

Comment:
I think that the current format is correct. because RFC8259 shows example like this.

(Page 12) `This is a JSON array containing two objects`
https://www.rfc-editor.org/rfc/rfc8259

Is there other reasons to change?

Reply:
Personal style :wink: I like to have this kind of code as small as possible. And I like it to be consistent with the go code.

1/5

Reply:
Got it. I wondered if there is a rule I do not know about the format of JSON.
I follow your style :+1: